src/notificationmanager.cpp

```diff
@@ -69,8 +70,50 @@ NotificationManager::NotificationManager(QObject* parent)
 
 NotificationManager::~NotificationManager()
 {
+    qDeleteAll(interfaces.values());
```

Comment:
This is unnecessary since `NotificationManager` is a parent of `QOfonoMessageWaiting` objects. But won't hurt much either.